scripts/tcl_commands/dft.tcl

```diff
@@ -0,0 +1,77 @@
+proc run_dft {args} {
+    if { $::env(DFT_ENABLE) } {
+        increment_index
+    }
+
+    if { ! [info exists ::env(RESET_PORT) ] && $::env(DFT_ENABLE) } {
+        if { ! [info exists ::env(CLOCK_PORT) ] } {
+            puts_info "::env(CLOCK_PORT) is not set"
+        }
+        puts_info "::env(RESET_PORT) is not set"
+        puts_warn "Skipping DFT"
+        set ::env(DFT_ENABLE) 0
+    }
+
+    if { $::env(DFT_ENABLE) } {
+        fault_chain \
+            -verilog $::env(synthesis_results)/$::env(DESIGN_NAME).v \
+            -liberty $::env(PDK_ROOT)/sky130A/libs.ref/sky130_fd_sc_hd/lib/sky130_fd_sc_hd__tt_100C_1v80.lib \
```

Comment:
Should probably be replaced with synth_lib?

Reply:
The whole path starting from the `$::env(PDK_ROOT)/....` right? Done